src/rgw/rgw_op.h

```diff
@@ -1065,7 +1065,7 @@ class RGWGetACLs : public RGWOp {
 
 class RGWPutACLs : public RGWOp {
 protected:
-  size_t len;
+  int len;
```

Comment:
i think `int` is reasonable for `RGWRestfulIO::recv_body()`, because it's either bytes_read or a negative error code. the alternative is to return `uint64_t` and allow errors to propagate as `rgw::io::Exception`s, but i worry that a lot of the callers are not exception safe (though there appear to be callers that aren't checking for errors here either)